packages/experimental-utils/src/ts-eslint/Rule.ts

```diff
@@ -165,6 +165,15 @@ type ReportDescriptor<TMessageIds extends string> =
   ReportDescriptorWithSuggestion<TMessageIds> &
     (ReportDescriptorNodeOptionalLoc | ReportDescriptorLocOnly);
 
+/**
+ * Plugins can add their settings using declaration
+ * merging against this interface.
+ */
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
```

Comment:
isn't this unnecessary now?
i'm surprised our lint rule didn't fire on this

Reply:
okay weird... none of the checks ran on your PR

Reply:
quite right, and yeah, that is weird...

I'll rebase to resolve the conflict and remove this at the same time, which'll hopefully trigger CI.